spring-cloud-stream-binder-kafka-streams/src/main/java/org/springframework/cloud/stream/binder/kafka/streams/QueryableStoreRegistry.java

```diff
@@ -44,10 +47,15 @@
 	public <T> T getQueryableStoreType(String storeName, QueryableStoreType<T> storeType) {
 
 		for (KafkaStreams kafkaStream : kafkaStreams) {
+			try{
 			T store = kafkaStream.store(storeName, storeType);
 			if (store != null) {
 				return store;
 			}
+			}
+			catch (InvalidStateStoreException ignored) {
+				//pass through
+			}
```

Comment:
Could you please fix the formatting with proper indentation?

Reply:
fixed